_includes/v22.2/sidebar-data/reference.json

```diff
@@ -250,6 +226,122 @@
                 "title": "<code>ALTER TABLE</code>",
                 "urls": [
                   "/${VERSION}/alter-table.html"
```

Comment:
I suggest omitting the `urls` block here and moving it down as a separate item within the `items` block to prevent all of those subcommands from being selected while the page is open.